Makefile

```diff
@@ -52,10 +52,19 @@ install_deps: $(GOPATH)/bin/glide
 KINESIS_STREAM_NAME ?= logs-test
 KINESIS_AWS_REGION ?= us-west-1
 
+# sets the `applicationName` in KCL consume.properties.
+# we want a dif application in Dev and Prod, so that they don't conflict
+# (e.g. app name determines the DynamoDB table that KCL uses to coordinate)
+APPLICATION_NAME ?= kinesis-to-firehose-dev
```

Comment:
(Same as https://github.com/Clever/kinesis-to-firehose/pull/1#discussion_r107053671)